homeassistant/components/sensor/fitbit.py

```diff
@@ -34,6 +34,7 @@
 
 CONF_MONITORED_RESOURCES = 'monitored_resources'
 CONF_CLOCK_FORMAT = 'clock_format'
+CONF_UNIT_SYSTEM = 'unit_system'
```

Comment:
This is already defined in `const.py`. Please import that.

Reply:
Ok, done